tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -0,0 +1,116 @@
+"""Test class for Content Hosts UI
+
+:Requirement: Content Host
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.constants import (
+    DISTRO_RHEL7,
+    FAKE_0_CUSTOM_PACKAGE,
+    FAKE_0_CUSTOM_PACKAGE_NAME,
+    FAKE_1_CUSTOM_PACKAGE,
+    FAKE_1_ERRATA_ID,
+    FAKE_1_YUM_REPO,
+    FAKE_2_CUSTOM_PACKAGE,
+    FAKE_2_CUSTOM_PACKAGE_NAME,
+)
+from robottelo.decorators import tier3
+from robottelo.products import (
+    YumRepository,
+    RepositoryCollection,
+    SatelliteToolsRepository,
+)
+from robottelo.vm import VirtualMachine
+
+
+@tier3
+def test_positive_end_to_end(session):
+    """Create all entities required for content host, set up host, register it
+    as a content host, read content host details, install package and errata.
+
+    :id: f43f2826-47c1-4069-9c9d-2410fd1b622c
+
+    :expectedresults: content host details are the same as expected, package
+        and errata installation are successful
+
+    :CaseLevel: System
+    """
+    org = entities.Organization().create()
+    lce = entities.LifecycleEnvironment(organization=org).create()
+    repos_collection = RepositoryCollection(
+        distro=DISTRO_RHEL7,
+        repositories=[
+            SatelliteToolsRepository(),
+            YumRepository(url=FAKE_1_YUM_REPO)
+        ]
+    )
+    repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
+    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+        repos_collection.setup_virtual_machine(vm)
+        result = vm.run('yum -y install {0}'.format(FAKE_1_CUSTOM_PACKAGE))
+        assert result.return_code == 0
+        with session:
+            session.organization.select(org.name)
+            # Ensure content host is searchable
+            assert session.contenthost.search(
+                vm.hostname)[0]['Name'] == vm.hostname
+            chost = session.contenthost.read(vm.hostname)
+            # Ensure all content host fields/tabs have appropriate values
+            assert chost['details']['name'] == vm.hostname
+            assert (
+                chost['details']['content_view'] ==
+                repos_collection._setup_content_data['content_view']['name']
```

Comment:
access to private attribute, I have created a property in this PR https://github.com/SatelliteQE/robottelo/pull/6218/files#diff-002cc267b83726c3b9cc934babf4a2a3R587
not merged yet

Reply:
you can you create that property , and one of us should rebase :)